support-services/src/main/scala/com/gu/support/catalog/CatalogService.scala

```diff
@@ -14,6 +14,34 @@ object CatalogService {
 }
 
 class CatalogService(val environment: TouchPointEnvironment, jsonProvider: CatalogJsonProvider) extends LazyLogging {
+
+  private def getRatePlanId(billingPeriod: BillingPeriod, fulfilmentOptions: FulfilmentOptions)
```

Comment:
`private` allows cross instance method access, meaning all instances of CatalogueService can access each other's private methods. I think what you want here is `private[this]`, which is a cheaper inline accessor.